lib/fragments-import-paths.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const parseImports = require('./parse-imports');
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Resolve the paths of the files of all the imports from each file visited stemming from the provided source.
+ *
+ * The `resolver` function will be called with a file path and an object with "basedir"
+ *
+ * resolveStatus is an object that contains "path" and "error". If the "error" property has an Error value, that means
+ * there was an issue when trying to resolve the path
+ *
+ * @param {string} source - Source code of .grpahql file to find import statements in
+ * @param {string} sourceLocation - File path of source
+ * @param {function} resolver - This function will be used to resolve the path for the file of an import
+ * @param {number} maxLevel - Go this many levels deep into searching for imports
+ * @returns {array<object>} - whose keys are `commentNode` and `resolveStatus` where resolveStatus is described above
+ */
+module.exports = function getAllImportedFiles(
```

Comment:
as discussed out of band, I think what we want is something more like  what we have in https://github.com/stefanpenner/graphql-fragment-import/blob/0cf0a69ccf06979f44f6b96119abc1f520403518/eslint-plugin/rules/validate-imports.js#L188-L200

i.e. to re-use [lineToImports](https://github.com/stefanpenner/graphql-fragment-import/blob/0cf0a69ccf06979f44f6b96119abc1f520403518/lib/inline-imports.js#L90-L104) and [getFragmentDefinitionsFromSource](https://github.com/stefanpenner/eslint-ast/blob/c25b6c1910ed9100c2ee1895737a2d0de687a28a/projects/graphql/eslint-plugin-graphql/parser.js#L131-L140) to build up a dictionary of `<importLine, fragments>` where `fragments` is either a list of `FragmentDefinition` nodes, or **possibly** a dictionary of fragment names to `FragmentDefinition` nodes.